.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -8,12 +8,16 @@ labels: ''
 assignees: ''
 
 ---
+I have read check documentation: https://checkstyle.org/config_xxxxxx.html#NameOfAffectedCheck: Yes/No
```

Comment:
remove all `: Yes/No`

`I have read check documentation: https://checkstyle.org/config_xxxxxx.html#NameOfAffectedCheck` already statement.
Pressing "Submit new issue" in UI is like signing a form.

Reply:
Done